gini/gini-vision-lib-ios
# Feature move tests to subspec

### Description
Moved all GiniVision tests into a test subspec. The tests related to the Example app have been kept in the same target (renamed from `GiniVision_Tests` to `Example_Tests`).
Some method's signature has been also improved.

### How to test
Run the job in Jenkins and check that there are two test target running (one for the GiniVision librbary and the other for the Example app).

### Merging
Automatic
